jmix-reports/reports/src/main/java/io/jmix/reports/ReportSecurityManager.java

```diff
@@ -113,10 +113,11 @@ public void applyPoliciesByEntityParameters(LoadContext lc, @Nullable MetaClass
      * @param screenId            - id of the screen
      * @param user                - caller user
      * @param inputValueMetaClass - meta class of report input parameter
-     *
+     * @param sort                - sorting type for the reports list
      * @return list of available reports
      */
-    public List<Report> getAvailableReports(@Nullable String screenId, @Nullable UserDetails user, @Nullable MetaClass inputValueMetaClass) {
+    public List<Report> getAvailableReports(@Nullable String screenId, @Nullable UserDetails user,
```

Comment:
You cannot change public API like this, it's breaking changes. Add a new method with `Sort` parameter and call it from all one. Also it will let you avoid passing `null` explicitly